docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -98,7 +98,7 @@ We used a [`jetson` board](/components/board/jetson/), but you can use any model
 ![Configuration of a jetson board with digital interrupts in the Viam app config builder.](/tutorials/navigate-with-rover-base/board-config-builder.png)
 
 2. Configure [digital interrupts](/components/board/#digital_interrupts) on your board to signal precise GPIO state changes to the [encoders](/components/encoder/) on your rover base.
-   Copy and paste the following into your board's **Attributes** to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
+   Copy and paste the following into your board's attributes field to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```

Comment:
```suggestion
   Find your board on the **CONFIGURE** tab in **Builder** mode. Click the **{}** (Switch to advanced) button on the right side of your board's card to switch to JSON attributes editing mode.
   Copy and paste the following JSON into your board's attributes field to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```
This changed slightly with the new board cards and it's now harder to find where to do it